images/linux/scripts/installers/1604/go.sh

```diff
@@ -12,25 +12,34 @@ source $HELPER_SCRIPTS/document.sh
 #   $2=MajorAndMinorVersion (1.11.1)
 #   $3=IsDefaultVersion (true or false)
 function InstallGo () {
-    curl -sL https://dl.google.com/go/go$2.linux-amd64.tar.gz -o go$2.linux-amd64.tar.gz
+    pattern="refs/tags/go$1([.0-9]{0,3})$"
+
+    ref=$( echo ./tmp/output.json | jq --arg pattern "$pattern" '[.[] | select( .ref | test($pattern))] | .[-1] | .ref' )
```

Comment:
Move `[.[] | select( .ref | test($pattern))] | .[-1] | .ref` to separate variable